exercises/custom-set/custom_set_test.rb

```diff
@@ -3,114 +3,303 @@
 require 'minitest/autorun'
 require_relative 'custom_set'
 
+# rubocop:disable Metrics/LineLength
```

Comment:
Hmm, I don't like this. The offending lines seem overly verbose. Would it be advisable to shorten them up in the x-common json?